translations/de.yaml

```diff
@@ -298,8 +286,8 @@ ember-gwr:
       createEntry: |-
         {modelName} erfassen
         {model, select,
-          project {}
-          other {(Neubau)}
+          building {(Neubau)}
+          other {}
```

Comment:
What's this change for?

Reply:
Leave Neubau for dwelllings and entrances too

Reply:
Done